lottie/build.gradle

```diff
@@ -54,25 +50,7 @@ dependencies {
   androidTestImplementation libs.androidx.test.espresso
 }
 
-task sourcesJar(type: Jar) {
-  from android.sourceSets.main.java.srcDirs
-  archiveClassifier.set('sources')
-}
-
-task javadoc(type: Javadoc) {
-  source = android.sourceSets.main.java.srcDirs
-  configurations.implementation.setCanBeResolved(true)
-  classpath += project.files(android.getBootClasspath().join(File.pathSeparator)) + configurations.implementation
-}
-
```

Comment:
I checked `publish` task dependencies, and it seemed like these two weren't actually used 🤷 

There is `sourceReleaseJar` and `javaDocReleaseJar` used respectively instead. I assume they are configured by vanniktech's plugin?